lib/ansible/module_utils/network/iosxr/argspec/acls/acls.py

```diff
@@ -0,0 +1,644 @@
+                                                },
+                                                'host_report': {
+                                                    'type': 'bool'
+                                                },
+                                                'mtrace': {
+                                                    'type': 'bool'
+                                                },
+                                                'mtrace_response': {
+                                                    'type': 'bool'
+                                                },
+                                                'pim': {
+                                                    'type': 'bool'
+                                                },
+                                                'trace': {
+                                                    'type': 'bool'
+                                                }
+                                            },
+                                            'type': 'dict'
+                                        },
+                                        'tcp': {
+                                            'options': {
+                                                'ack': {
+                                                    'type': 'bool'
+                                                },
+                                                'established': {
+                                                    'type': 'bool'
+                                                },
+                                                'fin': {
+                                                    'type': 'bool'
+                                                },
+                                                'psh': {
+                                                    'type': 'bool'
+                                                },
+                                                'rst': {
+                                                    'type': 'bool'
+                                                },
+                                                'syn': {
+                                                    'type': 'bool'
+                                                },
+                                                'urg': {
+                                                    'type': 'bool'
+                                                }
+                                            },
+                                            'type': 'dict'
+                                        }
+                                    },
+                                    'type': 'dict'
+                                },
+                                'remark': {
+                                    'type': 'str'
+                                },
+                                'sequence': {
+                                    'type': 'int'
```

Comment:
If its always required, we can have " required: True" for this, otherwise it's fine.

Reply:
We can have an ACL without specifying the sequence no. In that case the appliance itself with append the ACL with a sequence number which a +10 increment of the last one.

Reply:
@NilashishC  Then its perfectly fine.